algebras/algebra/src/test/scala/endpoints/algebra/client/JsonTestSuite.scala

```diff
@@ -17,6 +17,7 @@ trait JsonTestSuite[T <: JsonTestApi] extends ClientTestBase[T] {
         val addressStr = """{"street":"avenue1","city":"NY"}"""
         wireMockServer.stubFor(
           post(urlEqualTo("/user"))
+            .withHeader("Content-Type", containing("application/json"))
```

Comment:
Does that make the test fail if the client doesn’t supply the header?

Reply:
Yeah, I think so.